cartographer_ros/cartographer_ros/sensor_bridge.cc

```diff
@@ -73,6 +73,17 @@ void SensorBridge::HandleOdometryMessage(
   }
 }
 
+void SensorBridge::HandleNavSatFixMessage(
+    const std::string& sensor_id, const sensor_msgs::NavSatFix::ConstPtr& msg) {
+  LOG(FATAL) << "TODO(spielawa / wohe): NavSatFix support not yet implemented.";
+  const carto::common::Time time = FromRos(msg->header.stamp);
```

Comment:
Remove this and the following lines. It will now crash when it reaches the `LOG(FATAL)`.